notebooks/data-sources/TestGrid/metrics/build_pass_failure.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
<p>do you want to change the nomenclature in the parameter here as well?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/267/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
<p>good catch! updated the parameters as well!</p>